core/src/main/java/hudson/model/FingerprintCleanupThread.java

```diff
@@ -118,7 +118,7 @@ private boolean check(File fingerprintFile, TaskListener listener) {
             } else {
                 if (!fp.isAlive()) {
                     FingerprintFacet deletionBlockerFacet = fp.getFacetBlockingDeletion();
-                    listener.getLogger().println(deletionBlockerFacet.getClass().getName() + " created on " + DATE_CONVERTER.toString(deletionBlockerFacet.getTimestamp()) + " blocked deletion of " + fingerprintFile);
+                    listener.getLogger().println(deletionBlockerFacet.getClass().getName() + " created at timestamp: " + deletionBlockerFacet.getTimestamp() + " blocked deletion of " + fingerprintFile);
```

Comment:
Wouldn't this be much nicer?

```suggestion
                    listener.getLogger().println(deletionBlockerFacet.getClass().getName() + " created on " + new Date(deletionBlockerFacet.getTimestamp()) + " blocked deletion of " + fingerprintFile);
```

(Assuming this is millis. Untested.)